lisa/utils.py

```diff
@@ -64,6 +65,20 @@ def get_logger(cls, suffix=None):
             name += '.' + suffix
         return logging.getLogger(name)
 
+class Subclassable:
```

Comment:
What about a static module method that we could import?

To me it feels simpler that way - if you want to get the subclasses of a given class you can just import & call `get_subclasses(cls)` instead of having to understand you need to inherent from this special class.

Reply:
> static module method
You mean a plain function ;-) ?

Indeed, that can be a plain function in utils.py . I probably wrote that this way as an "extension" of `__subclasses__()`, but there is no real reason.